src/amo/pages/Addon/index.js

```diff
@@ -545,18 +546,10 @@ export class AddonBase extends React.Component {
                   />
                 )}
                 {showGetFirefoxButton && (
```

Comment:
can you just check for addon here instead of this `showGetFirefoxButton` since you do the compatibility check in `GetFirefoxButton` now, right?

Reply:
Yeah, that should work. Thanks!